packages/@aws-cdk/aws-iotevents/lib/expression.ts

```diff
@@ -12,6 +12,20 @@ export abstract class Expression {
     return new StringExpression(value);
   }
 
+  /**
+   * Create a expression as String from the given string.
+   */
+  public static asString(value: string): Expression {
```

Comment:
Currently, when users want to use literal string in the expression, it is needed to use `fromString` with double-quotes as following:
```ts
fromString('"my-data"')
```
And when users want to use literal number, it is needed to use `fromString` with string as following:
```ts
fromString('12.5')
```

Actually, I want to provide `fromString` and `fromNumber` as following:
```ts
fromString('my-data') // double-quotes is not needed
fromNumber(12.5) // apply number instead of string
```

But already I have used `fromString` name space...
`literalString` and `literalNumber` are better?

Reply:
In this case, the `iotevents-actions` package seems to be experimental, which means we are *technically* allowed to break its behavior in breaking ways if we figure out we made a mistake in the past (that's what the point of that stability is).

So I'd prefer for us to keep the prime real estate names, and change the behavior of `fromString` and `fromNumber` to have the desired behavior.

If you are concerned about inconveniencing users, fair enough and I love that instinct. We can choose to do one of the following things:

- Introduce a feature flag to gate the old vs new behavior of the `fromString()` method.
- Introduce a new method (immediately `@deprecate`d) named `legacyFromString()` or something, and clearly explain in the PR body that users relying on the old behavior should either fix their code or switch to the other method.

Reply:
@rix0rrr 
Thanks for your suggestion!
I have reverted the commit for `asString` and `asNumber`. And I will implement `fromString` and `fromNumber` as breaking change in another PR. Because it will have small changing some tests already written.

Reply:
I'm creating the PR https://github.com/yamatatsu/aws-cdk/pull/30 for breaking change of the behavior of `Expression.fromString()`.